mgmt_cli_test.py

```diff
@@ -44,7 +44,7 @@ def test_mgmt_repair_nemesis(self):
             1) Run cassandra stress on cluster.
             2) Add cluster to Manager and run full repair via Nemesis
         """
-        self._generate_load()
```

Comment:
seems like those were on purpose running while the repair is happening, why it should fail the test ?

Reply:
_generate_load is set to run for 5 seconds... it is not the case that the stress will be running during the whole test anyway